sdk/lib/stellar/client.rb

```diff
@@ -199,7 +199,7 @@ def submit_transaction(tx_envelope:, options: {skip_memo_required_check: false})
       unless options[:skip_memo_required_check]
         check_memo_required(tx_envelope)
       end
-      @horizon.transactions._post(tx: Base64.encode64(tx_envelope.to_xdr))
+      @horizon.transactions._post(tx: Base64.strict_encode64(tx_envelope.to_xdr))
```

Comment:
```suggestion
      @horizon.transactions._post(tx: tx_envelope.to_xdr(:base64))
```

I think we could simplify this, since `ruby-xdr` has an option to serialize into base64, and it correctly uses `strict_encode64`: https://github.com/astroband/ruby-xdr/blob/master/lib/xdr/concerns/converts_to_xdr.rb#L50.

Reply:
Makes a lot of sense to me!

Reply:
I never expected to get a good review on such a small diff :joy: Well done!